src/components/Global/BaseDialog.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-dialog v-model="show" persistent max-width="650" :scrollable="scrollable" elevation scrim="surface-1">
+    <v-card :id="id" class="dialog-card" color="surface-1">
+      <v-card-title class="d-flex justify-space-between pt-4">
+        <h1 v-if="title">{{ title }}</h1>
+        <slot name="title" />
+      </v-card-title>
+      <v-card-text>
+        <slot name="body" />
+      </v-card-text>
+      <div class="dialog-actions d-flex justify-end">
+        <slot name="actions" />
+      </div>
```

Comment:
I had this wrapped in a v-card-actions initially but it did weird stuff with the padding so I just used a div instead